assets/tests/components/routeLadders.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+import renderer from "react-test-renderer"
+import RouteLadders from "../../src/components/routeLadders"
+import { Route, TimepointsByRoute } from "../../src/skate"
+
+const dispatch = () => undefined
+
+test("renders a route ladder", () => {
+  const routes: Route[] = [{ id: "1" }, { id: "28" }]
+  const timepointsByRoute: TimepointsByRoute = {
+    "1": [
+      { id: "WASMA" },
+      { id: "MELWA" },
+      { id: "HHGAT" },
+      { id: "MAPUT" },
+      { id: "CNTSQ" },
+      { id: "MIT" },
+      { id: "HYNES" },
+      { id: "MASTA" },
+      { id: "DUDLY" },
+    ],
+    "28": [
+      { id: "MATPN" },
+      { id: "WELLH" },
+      { id: "MORTN" },
+      { id: "HRUGG" },
+      { id: "BLTAL" },
+      { id: "ROXBS" },
+      { id: "FRNPK" },
+      { id: "GHALL" },
+      { id: "LATAC" },
+      { id: "WARWL" },
+      { id: "DUDLY" },
+      { id: "LOUIS" },
+      { id: "MALCX" },
+      { id: "RUGG" },
```

Comment:
We don't need this much sample data. 2 or 3 for each route would be fine. Using less data would make it easier to add fields to the timepoint type, and easier to scroll through this file once it has lots of tests.

(Same for `routeLadder.test.tsx`)